src/dinglehopper/ocrd-tool.json

```diff
@@ -1,17 +1,13 @@
 {
   "version": "0.9.7",
   "git_url": "https://github.com/qurator-spk/dinglehopper",
+  "dockerhub": "ocrd/dinglehopper",
```

Comment:
Not sure if this should fly in the quratorspk namespace instead, but leaving it for now...

Reply:
I updated #121 accordingly and I'll leave that in for now.